index.bs

```diff
@@ -5583,6 +6045,25 @@ partial dictionary MLOpSupportLimits {
         - an {{MLOperand}}. The output tensor of the same shape as *input*.
 </div>
 
+<table class='data' link-for="MLGraphBuilder/prelu(input, slope, options)">
+  <thead class=stickyheader>
+    <tr>
+      <th>input operand</th>
+      <th>[=/allowed data types=]</th>
+      <th>[=/allowed ranks=]</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>{{input}}</td>
+    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}, {{MLOperandDataType/"int8"}}</td>
+    <td>[=/any rank|N=]</td>
+  <tr>
+    <td>{{slope}}</td>
+    <td>[=/same as=] {{input}}</td>
+    <td>[=/any rank|N=]</td>
```

Comment:
Should slope's rank in the range [0, input rank]? Slope's shape must be bidirectionally broadcastable to the shape of input.

Reply:
4bf14f6504ba0587d33ee4439cc5e05f7f081085 updates allowed ranks to be a range... but given that input and slope are **bidirectionally** broadcast, can't _slope_'s rank be larger than _input_'s?

Reply:
> but given that input and slope are bidirectionally broadcast, can't slope's rank be larger than input's?

You are correct, slope's rank can be larger than input's. I made a mistake. The original "any rank" is fine. Apologize for that.